src/server/game/DataStores/DB2Structure.h

```diff
@@ -2065,11 +2065,11 @@ struct RewardPackEntry
 {
     uint32 ID;
     uint32 Money;
-    float ArtifactPowerMultiplier;
-    uint8 ArtifactQuestXPIndex;
-    uint8 ArtifactCategoryID;
+    float ArtifactXPMultiplier;
+    uint32 ArtifactXPDifficulty;
```

Comment:
but you shouldnt have changed the types, only names (i just copypasted these lines without thinking from quest code)